app/experimenter/nimbus-ui/src/components/HeaderExperiment/index.tsx

```diff
@@ -80,11 +80,9 @@ const HeaderExperiment = ({
 
         <p className="header-experiment-status position-relative mt-2 d-inline-block">
           <StatusPill label="Draft" active={status.draft} />
-          {
-            /* istanbul ignore next until EXP-866 final */ status.preview && (
-              <StatusPill label="Preview" active={status.preview} />
-            )
-          }
+          {status.preview && (
+            <StatusPill label="Preview" active={status.preview} />
```

Comment:
I think `active={status.preview}` could just be `active` since it's already wrapped in that same condition.